server/zanata-frontend/src/frontend/app/styles/style.less

```diff
@@ -61,7 +61,6 @@ body.templatestyle {
   align-items: flex-start;
   -webkit-align-items: flex-start; /* Safari 7.0+ */
   display: flex;
-  display: -webkit-box;      /* OLD - iOS 6-, Safari 3.1-6 */
   display: -webkit-flex;
```

Comment:
removed every instance of this to ensure we don't come across any other css problems